crates/swc_ecma_minifier/src/compress/optimize/sequences.rs

```diff
@@ -2414,3 +2449,16 @@ pub(crate) fn is_trivial_lit(e: &Expr) -> bool {
         _ => false,
     }
 }
+
+/// This assumes `a.left.to_id() == b.left.to_id()`
+fn can_drop_op_for(a: AssignOp, b: AssignOp) -> bool {
+    if a == op!("=") {
+        return true;
+    }
+
+    if a == b {
+        return matches!(a, op!("+=") | op!("*="));
```

Comment:
This is just looking for associative property? We should be able to merge `/=`, etc?

Reply:
This first one should be prefixed with `-` so I just excluded it